### DISCUSSION TITLE
🎉 Introducing the Bevy Input Mapper Plugin.

### DISCUSSION THREAD

**Discussion by atahabaki** [bevyengine/bevy#9728] [Show and tell]
I'm thrilled to introduce the Bevy Input Mapper Plugin, a tool that simplifies input mapping in Bevy game development. I'm passionate about its potential to enhance the Bevy game engine development experience.

Here are the planned features:

- **Load Input Bindings from Files:** In the pipeline is the ability to load input bindings from files, streamlining the configuration process.

- **Customizable Scenarios:** I'm planning to introduce scenario profiles, allowing you to create and customize input configurations for specific in-game scenarios like driving, walking, and fighting.

I hope that this plugin will find a valuable place within the Bevy game engine, and I'm excited to bring these enhancements to the Bevy community. Explore the plugin and stay updated on its progress in the [GitHub repository](https://github.com/atahabaki/bevy_input_mapper). 🚀🎮

**Comment by rparrett** [comment]
Cool. Don't forgot to add it to [Bevy Assets](https://bevyengine.org/assets/).
